mindsdb/integrations/handlers/flaml_handler/requirements.txt

```diff
@@ -1,2 +1,2 @@
 flaml<=1.2.3
-dill
+type-infer
```

Comment:
`type-infer` is part of our ML packages. I think we probably want to make it part of our defaults given we're recommending people to actively use it when developing new ML engines.

It is fairly light in deps ([source](https://github.com/mindsdb/type_infer/blob/staging/pyproject.toml)), but if you think there are any here that would be problematic we can keep it as optional.

Reply:
We're trying to only have things in the main requirements.txt that are used in the core codebase. That said, with the list of default handlers installed when you install `mindsdb`, it should get installed in most cases anyway

Reply:
Okay, let's leave it out of default/core reqs 👍